src/AwesomeCMSCore/AwesomeCMSCore/Extension/ServiceCollectionExtensions.cs

```diff
@@ -161,7 +162,7 @@ public static IServiceCollection InjectApplicationServices(this IServiceCollecti
         public static IServiceCollection AddCustomAuthentication(this IServiceCollection services)
         {
             services
-                .AddIdentity<User, ApplicationRole>()
+                .AddIdentity<User, IdentityRole>()
```

Comment:
Dont change to IdentityRole. The ApplicationRole is already inherit from IdentityRole
You can see from here https://github.com/Awesome-CMS-Core/Awesome-CMS-Core/blob/master/src/AwesomeCMSCore/Modules/AwesomeCMSCore.Modules.Entities/Entities/ApplicationRole.cs

Reply:
I see already. But After Group-based permission not necessary for ApplicationRole

Reply:
Okay. I will using after that

Reply:
You can see UserService for more detail